pkg/endpoint/bpf.go

```diff
@@ -940,20 +941,57 @@ func (e *Endpoint) e.addPolicyKey(keyToAdd policy.Key, entry policy.MapStateEntr
 		e.getLogger().WithError(err).Errorf("Failed to add PolicyMap key %s %d", policyKeyToPolicyMapKey.String(), entry.ProxyPort)
 		return err
 	}
-	
+
 	// Operation was successful, add to realized state.
 	e.realizedPolicy.PolicyMapState[keyToAdd] = entry
 
 	return nil
 }
 
+func (e *Endpoint) proxyID(key policy.Key) string {
+	return policy.ProxyID(e.ID, key.TrafficDirection == trafficdirection.Ingress.Uint8(), u8proto.U8proto(key.Nexthdr).String(), key.DestPort)
+}
+
+func (e *Endpoint) applyPolicyMapChanges() error {
+	errors := []error{}
+
+	adds, deletes := e.realizedPolicy.PolicyMapChanges.GetMapChanges()
+
+	e.getLogger().Debugf("applyPolicyMapChanges: adds: %v, deletes: %v", adds, deletes)
```

Comment:
Please use withfields here instead of printing the adds and deletes in the string.

Reply:
Deleted this, was too noisy.